### DISCUSSION TITLE
Is there a way to hide buffers in the buffers component?

### DISCUSSION THREAD

**Discussion by vdolbilov** [nvim-lualine/lualine.nvim#619] [Q&A]
Hi, I have my tabline configured to show the "buffers" component, and I want to exclude some buffers from showing up there at all, like NvimTree for instance. I can set the value under "filetype_names" to "" but that just shows a blank buffer. Is there a way to hide a buffer completely?

**Comment by phi-mah** [comment]
This discussion is quite old, but today I made #887 which does exactly this
